tests/fixtures/orderforms/mip.json

```diff
@@ -124,7 +124,7 @@
       "source": "tissue (fresh frozen)",
       "status": "affected",
       "subject_id": "subjectsample4",
-      "synopsis": "",
+      "synopsis": "H\u00e4r kommer det att komma en v\u00e4ldigt l\u00e5ng text med f\u00f6r synopsis.",
```

Comment:
Looks strange! Is it supposed to be like this?

Reply:
All other samples in the fixture has that synopsis, so i just copied it, but maybe we can change it to english to make it a bit prettier?